agent/app/agent.go

```diff
@@ -579,3 +588,24 @@ func (agent *ecsAgent) startACSSession(
 	seelog.Critical("ACS Session handler should never exit")
 	return exitcodes.ExitError
 }
+
+// validateRequiredDockerVersion validates docker version to be minimum 1.9.0
+func (agent *ecsAgent) validateRequiredDockerVersion(selector string) error {
```

Comment:
Why can't we use a combination of ACS capabilities and API versions like we have done for other features?

Reply:
talked offline